tools/recording/record_common.py

```diff
@@ -14,6 +14,7 @@
 
 logger = logging.getLogger(__name__)
 
+SPECIAL_SAMPLING_OPS = ['conv2d','linear']
```

Comment:
Add some explanation for what this variable means

Reply:
added more comments